styles.css

```diff
@@ -56,6 +57,10 @@ header {
   background-color: #acb7c3;
 }
 
+.desktop-nav {
+  display: none;
+}
```

Comment:
- [ ] I observed that you repeated this line of code a lot, this is not the best according to CSS best practices. Kindly implement it the way you did in line 189 -190 for all the repeated code. 👍
**Note: Please kindly look through lines (147, 172, 190, 406, 472, 566) of code carefully. 👍**